src/config.js

```diff
@@ -0,0 +1,11 @@
+//const env = process.env.NODE_ENV; // 'dev' or 'test'
+
+const config = {
+  host: `http://localhost:8081`,
```

Comment:
This should not be called host, host is only `localhost` part. https://url.spec.whatwg.org